.github/workflows/scripts/docs/build-docs.sh

```diff
@@ -10,7 +10,7 @@ docker run \
   --user "$(id -u):$(id -g)" \
   --volume "${PWD}:/docs" \
   --name "build-docs" \
-  squidfunk/mkdocs-material:9.2 build --strict
+  squidfunk/mkdocs-material:9.5.3 build --strict
```

Comment:
We use `Major.Minor` only here to get the patch updates for free (unless you already have the image locally, then you'd need to force pull). We do this mainly for the benefit of CI since this isn't updated that often (no dependabot integration).

```suggestion
  squidfunk/mkdocs-material:9.5 build --strict
```